ftw/labels/migrations/migrate.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from ftw.labels.interfaces import ILabelJar
+from ftw.labels.interfaces import ILabelJarChild
+from ftw.labels.interfaces import ILabelRoot
+from ftw.labels.interfaces import ILabelSupport
+from ftw.labels.labeling import ILabeling
+from plone import api
+from zope.annotation.interfaces import IAnnotations
+
+
+# The migration function -------------------------------------------------------
```

Comment:
This is not needed, it's clear enough without these decorative comments. Please ✂️